pkg/noderesourcetopology/filter.go

```diff
@@ -181,3 +207,22 @@ func resourceFoundOnNode(resName v1.ResourceName, wantQuantity resource.Quantity
 	}
 	return false
 }
+
+func allocateFromNUMA(nodes NUMANodeList, nodeID int, container v1.Container) {
```

Comment:
Does `numaID` better than `nodeID`?

Reply:
Thank you for the review.
I wanted to keep consistency with the first argument which is ```nodes``` but I presume ```numaID``` is clearer indeed.